presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -167,6 +167,7 @@ Property Name                                      Description
 
 ``security.config-file``                           Path of config file to use when ``hive.security=file``.
                                                    See :ref:`hive-file-based-authorization` for details.
+``hive.non-managed-table-writes-enabled``          Enable writes to non-managed Hive tables.                    ``false``
```

Comment:
I prefer keeping it as it's more descriptive this way. Regarding consistency, there are many other hive config parameters that end with `enabled` and there are some (like bucketing-related parameters) that do not, so we have a mix of both.